src/sql/parts/notebook/outputs/factories.ts

```diff
@@ -0,0 +1,94 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+import * as widgets from './widgets';
+import { IRenderMime } from './common/renderMimeInterfaces';
+
+/**
+ * A mime renderer factory for raw html.
+ */
+export const htmlRendererFactory: IRenderMime.IRendererFactory = {
+    safe: true,
+    mimeTypes: ['text/html'],
+    defaultRank: 50,
+   createRenderer: options => new widgets.RenderedHTML(options)
+};
+
+/**
+ * A mime renderer factory for images.
+ */
+export const imageRendererFactory: IRenderMime.IRendererFactory = {
+    safe: true,
+    mimeTypes: ['image/bmp', 'image/png', 'image/jpeg', 'image/gif'],
+    defaultRank: 90,
+    createRenderer: options => new widgets.RenderedImage(options)
+};
+
+// /**
+//  * A mime renderer factory for LaTeX.
+//  */
+// export const latexRendererFactory: IRenderMime.IRendererFactory = {
```

Comment:
Can you create a task for each renderer we disabled? We should start tracking here since they're built-in.

Reply:
Created a task #3155